src/Subscription/Subscription.Test/ScenarioTests/SubscriptionTests.ps1

```diff
@@ -18,37 +18,36 @@ Create subscription
 #>
 function Test-UpdateRenameSubscription
 {
-    $subId = "bc085fce-1a23-4734-b588-7c36b622317e"
```

Comment:
`TestUpdateCancelSubscription` in `SubscriptionTests.cs` is removed.

Reply:
In this case, the subscription is not available for the cancel operation since the subscription has associated resources. So, I commented out this.